week2/materials/homework/webapp/src/components/Products.js

```diff
@@ -1,15 +1,29 @@
 import useProducts from "../hooks/useProducts";
+import useNotifications from "../hooks/useNotification";
 
 function Products() {
-  const { products, cart, addProduct, removeProduct } = useProducts();
+  const { products, cart, addProduct, Total, removeProduct } = useProducts();
```

Comment:
```suggestion
  const { products, cart, addProduct, total, removeProduct } = useProducts();
```
camelCase for variables 🙏